app/tailwind.config.ts

```diff
@@ -1,13 +1,63 @@
 import type { Config } from "tailwindcss";
+import path from 'node:path'; 
 
 const config: Config = {
   content: [
     "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/**/*.{ts,tsx}",
+    path.join(path.dirname(require.resolve('@coinbase/onchainkit')), '**/*.js'), 
   ],
   theme: {
-    extend: {},
+    fontFamily: {
+      sans: ['Inter', 'sans-serif'],
+    },
+    extend: {  
+      fontFamily: {  
+        display: 'DM Sans, sans-serif', // Set to font of your choice
```

Comment:
Current EIS using Robot font, can you modify to set the font properly?

Reply:
Sure. I'll replace it to Robot.